### PR TITLE
[flutter_conductor] auto-generate PR title and description via query params

### PR DESCRIPTION
**Pull request by christopherfujino**
Set the PR title and body description in the query params of the new PR link the conductor prints out.

part of https://github.com/flutter/flutter/issues/81491

I generated this test PR: https://github.com/flutter/flutter/pull/87312
With the following link: https://github.com/flutter/flutter/compare/flutter-2.5-candidate.1...christopherfujino:master?expand=1&title=%5Bflutter_releases%5D+Flutter+dev+2.5.0-1.0.pre+Framework+Cherrypicks&body=%23+Flutter+dev+2.5.0-1.0.pre+Framework%0A%0A%23%23+Scheduled+Cherrypicks%0A%0A-+commit%3A+c451b6b0d7a1148a1ce56d476dd57eaab386223f

### CODE REVIEW

**Comment by christopherfujino** [comment]
- commit: 6185ae9005ca7dfce3622ce1e4688c729d342b56
- commit: 33fd4f15a9addd260fe92a8456b6f0cca12c1c36

File: dev/conductor/lib/globals.dart [hunk 1]
```diff
@@ -7,6 +7,8 @@ import 'package:file/file.dart';
 import 'package:file/local.dart';
 import 'package:platform/platform.dart';
 
+import './proto/conductor_state.pb.dart' as pb;
```
Comment (jmagman) [inline dev/conductor/lib/globals.dart:10]:
```suggestion
import 'proto/conductor_state.pb.dart' as pb;
```
Or is there some reason to `./`?

**Review by jmagman — APPROVED** [review]
LGTM

File: dev/conductor/lib/globals.dart (hunk 1 above)
Comment (christopherfujino) [reply dev/conductor/lib/globals.dart]: hmm, no reason, done

**Pull request merged**